src/SUMMARY.md

```diff
@@ -15,6 +15,8 @@
     * [Firefox for iOS](products/firefox_ios/README.md)
         * [Adding a new locale](products/firefox_ios/adding_new_ios_locale.md)
         * [How to test](products/firefox_ios/testing.md)
+    * [Focus](products/focus/README.md)
+            * [How to test](products/focus/testing_focus.md)
```

Comment:
Fix the indentation here. Because of this, the page doesn't show up in the compiled book.